Doc/library/importlib.rst

```diff
@@ -124,6 +124,11 @@ Functions
     need to call :func:`invalidate_caches` in order for the new module to be
     noticed by the import system.
 
+    If the module cannot be imported, :func:`import_module` will raise an
+    :exc:`ImportError`. If the module is found and loaded, but the code in the
+    module raises an exception, :func:`import_module` will pass that exception
+    to the module that called it.
```

Comment:
Trying to incorporate everything discussed at the same time.

@brettcannon's  suggestion:
- is more specific on the type of exception that can be expected.
- does not incorporate the remark from @hauntsaninja, that criticises the fact that "cannot be imported" is very generic.

In the first thread @kj7rrv suggested a nice way of explaining how a found module can't be imported. Not sure if it's now too specific, but let's continue the discussion:

```suggestion
    If the module cannot be found or its files cannot be accessed,
    :func:`import_module` will raise :exc:`ImportError` or an appropriate
    subclass like :exc:`ModuleNotFoundError`.
```